ModelBase/src/persistence/NodeIdMap.h

```diff
@@ -44,6 +44,8 @@ class MODELBASE_API NodeIdMap {
 
 		void remove(const Node* node);
 
+		bool containsNode(const Node* node);
```

Comment:
this should be called contains (no need for node, since it's clear from the argument type)